mdw-common/src/com/centurylink/mdw/dataaccess/db/CommonDataAccess.java

```diff
@@ -208,6 +208,28 @@ protected String dateConditionToMySQL(String value) {
         return value;
     }
 
+    protected Long getElapsedTime(Long instanceId)throws SQLException{

```

Comment:
`getElapsedTime()` should take ownerType and instanceId.  Both of these should be used in the query.